payment-manager/src/main/java/com/qpay/paymentmanager/PaymentManagerApplication.java

```diff
@@ -10,4 +12,8 @@ public static void main(final String[] args) {
         SpringApplication.run(PaymentManagerApplication.class, args);
     }
 
+    @Bean
+    public RestTemplate restTemplate(){
+        return new RestTemplate();
+    }
```

Comment:
All beans should be created in config files, see https://www.geeksforgeeks.org/spring-configuration-annotation-with-example/
remember SRP (single responsibility principle)? Our main class is only responsible to run the app.